src/lib/components/AnnouncementCard/AnnouncementCard.svelte

```diff
@@ -38,7 +39,7 @@
 <a
 	href={linkHref}
 	target="_blank"
-	class="h-[450px] w-[300px] max-md:w-[140px] max-md:h-[222px] bg-white text-sucu-gray-dark rounded shadow-card-shadow hover:shadow-card-shadow-hover transition-shadow flex flex-col items-center"
+	class="${className} h-[450px] w-[300px] max-md:w-[140px] max-md:h-[222px] bg-white text-sucu-gray-dark rounded shadow-card hover:shadow-card-shadow-hover transition-shadow flex flex-col items-center"
```

Comment:
can use `cn` here for better readability

```suggestion
	class={cn(className, "h-[450px] w-[300px] max-md:w-[140px] max-md:h-[222px] bg-white text-sucu-gray-dark rounded shadow-card hover:shadow-card-shadow-hover transition-shadow flex flex-col items-center")}
>
```

Reply:
เอา className ออกละ มันไม่ได้ใช้